pkg/kubelet/kubelet_pods_test.go

```diff
@@ -605,6 +607,127 @@ func TestMakeEnvironmentVariables(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "configmap",
+			ns:   "test1",
+			container: &v1.Container{
+				Env: []v1.EnvVar{
```

Comment:
nit, since `EnvFrom` is processed before `Env`, it makes tests of this functionality much more readable to me if `EnvFrom` appears before `Env`.